apps/clock/js/alarm.js

```diff
@@ -175,6 +250,56 @@ define(function(require, exports, module) {
           this._notifyChanged(/* removed = */ true);
         });
       });
+    },
+
+    /** Sort all the alarms and return the next upcoming
+     *  alarm from the list
+     */
+    sortAlarms: function(storeAlarms) {
```

Comment:
This function should be named something a little more descriptive -- e.g. `getNextAlarm`